src/ya-comiste-backoffice/pages/products/create.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+
+import * as React from 'react';
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import sx from '@adeira/sx';
+
+export default function ProductsCreatePage(): React.Node {
+  return (
+    <Formik
+      initialValues={{
+        name: '',
+        description: '',
+      }}
+      validate={(values) => {
+        const errors = {};
+        if (!values.name) {
+          errors.name = 'Product name is required';
+        }
+        if (!values.description) {
+          errors.description = 'Product description is required';
+        }
+        return errors;
+      }}
+      onSubmit={(values, { setSubmitting }) => {
+        setTimeout(() => {
+          // eslint-disable-next-line no-undef,no-alert
+          alert(JSON.stringify(values, null, 2)); // TODO (graphql mutation)
+          setSubmitting(false);
+        }, 400);
+      }}
+    >
+      {({ isSubmitting }) => (
+        <Form>
+          <div className={styles('row')}>
+            <label htmlFor="name">Product name</label>
```

Comment:
the htmlFor should point to the input's id, not name. Or does Field set the id=name?

Another approach is to put the input element inside the label tag.

Reply:
> Or does Field set the id=name?

No, it does not. This is a mistake, thanks!